docs/rfc/rfc-012-abci++.md

```diff
@@ -0,0 +1,253 @@
+# RFC 012: ABCI++
```

Comment:
I already have a pending PR for 12. Are you really gonna kick me down the road again? 😆

Reply:
(updated)